src/modules/async.js

```diff
@@ -0,0 +1,37 @@
+const createGame = () => {
+  const game = fetch(
+    'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/',
+    {
+      method: 'POST',
+      headers: {
+        'Content-type': 'application/json',
+      },
+      body: JSON.stringify({ name: 'My cool new game' }),
+    },
+  );
+  return game
+    .json()
+    .then((data) => localStorage.setItem('Data', /\w{20}/.exec(data.result)[0]));
+};
+
+const post = async (gameId, data) => {
+  await fetch(
+    `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameId}/scores/`,
+    {
+      method: 'POST',
+      headers: {
+        'Content-type': 'application/json',
+      },
+      body: JSON.stringify(data),
+    },
+  );
```

Comment:
- [ ] Please wrap this in a try and catch block. This is to enable you to catch any errors that might occur when using async await 👍